src/Parser/DemonHunter/Vengeance/Modules/Statistics/Spells/EmpowerWards.js

```diff
@@ -1,30 +1,60 @@
 import React from 'react';
-
 import Analyzer from 'Parser/Core/Analyzer';
 import Combatants from 'Parser/Core/Modules/Combatants';
-
 import SPELLS from 'common/SPELLS';
 import SpellIcon from 'common/SpellIcon';
-
 import { formatPercentage, formatDuration } from 'common/format';
 import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+import SCHOOLS from 'common/MAGIC_SCHOOLS';
+
 
 class EmpowerWards extends Analyzer {
   static dependencies = {
     combatants: Combatants,
+    spellUsable: SpellUsable,
   };
 
-  statistic() {
-    const empowerWardsUptime = this.combatants.selected.getBuffUptime(SPELLS.EMPOWER_WARDS.id);
+  hitsWithNoWards = 0;
+  hitsWithWards = 0;
+  hitsWithWardsOffCD = 0;
 
-    const empowerWardsUptimePercentage = empowerWardsUptime / this.owner.fightDuration;
+  on_toPlayer_damage(event) {
+    // Physical
+    if (event.ability.type === SCHOOLS.ids.PHYSICAL) {
+      return;
+    }
+    if (this.combatants.selected.hasBuff(SPELLS.EMPOWER_WARDS.id, event.timestamp)) {
+      this.hitsWithWards += 1;
+    }else{
+      this.hitsWithNoWards += 1;
 
+      const isAvailable = this.spellUsable.isAvailable(SPELLS.EMPOWER_WARDS.id);
+      if(isAvailable) {
+        this.hitsWithWardsOffCD += 1;
+      }
+    }
+  }
+
+
+
+  statistic() {
+    const EMPOWER_WARDS_UPTIME = this.combatants.selected.getBuffUptime(SPELLS.EMPOWER_WARDS.id);
+    const EMPOWER_WARDS_UPTIME_PERCENTAGE = EMPOWER_WARDS_UPTIME / this.owner.fightDuration;
+    const HITS_WITH_WARDS_PERCENTAGE = this.hitsWithWards / (this.hitsWithWards + this.hitsWithNoWards);
     return (
       <StatisticBox
         icon={<SpellIcon id={SPELLS.EMPOWER_WARDS.id} />}
-        value={`${formatPercentage(empowerWardsUptimePercentage)}%`}
-        label="Empower Wards Uptime"
-        tooltip={`The Empower Wards total uptime was ${formatDuration(empowerWardsUptime / 1000)}.`}
+        value={`${formatPercentage(HITS_WITH_WARDS_PERCENTAGE)}%`}
+        label="Hits Mitigated By Empower Wards"
```

Comment:
Please don't unnecessarily capitalize words, only hits should be capitalized here.